app/src/main/java/com/limelight/binding/input/ControllerHandler.java

```diff
@@ -2104,7 +2104,16 @@ else if (deviceContext.vibrator != null) {
             else if (foundMatchingDevice && !vibrated && prefConfig.vibrateFallbackToDevice) {
                 // We found a device to vibrate but it didn't have rumble support. The user
                 // has requested us to vibrate the device in this case.
-                rumbleSingleVibrator(deviceVibrator, lowFreqMotor, highFreqMotor);
+
+                // We cast the unsigned short value to a signed int before multiplying by
+                // the preferred strength. The resulting value is capped at 65534 before
+                // we cast it back to a short so it doesn't go above 100%.
+                short lowFreqMotorAdjusted = (short)(Math.min(((lowFreqMotor & 0xffff)
+                        /100*prefConfig.vibrateFallbackToDeviceStrength), Short.MAX_VALUE*2));
+                short highFreqMotorAdjusted = (short)(Math.min(((highFreqMotor & 0xffff)
+                        /100*prefConfig.vibrateFallbackToDeviceStrength), Short.MAX_VALUE*2));
```

Comment:
We're losing some precision here due to integer divison.

Can you change to `((motor & 0xffff) * prefConfig.vibrateFallbackToDeviceStrength) / 100` instead?

Reply:
That's right, it's fixed. Thank you.